src/drivers/intel/hda/hda-dma.c

```diff
@@ -407,8 +407,7 @@ static int hda_dma_enable_unlock(struct dma_chan_data *channel)
 	}
 
 	/* start link output transfer now */
-	if (channel->direction == DMA_DIR_MEM_TO_DEV &&
-	    !(hda_chan->state & HDA_STATE_RELEASE))
+	if (channel->direction == DMA_DIR_MEM_TO_DEV)
```

Comment:
I'm probably missing something, I thought `DMA_DIR_MEM_TO_DEV` was only for link DMA playback?